packages/choreography-lint/rules/no-renaming-operator.js

```diff
@@ -0,0 +1,43 @@
+// no-renaming-operator.js: must NOT rename choreographic operators
+// https://github.com/shumbo/choreography-ts/issues/19
+
+module.exports = {
```

Comment:
I want those to be typed. 

https://typescript-eslint.io/developers/custom-rules#utils-package